src/data/queries/index.ts

```diff
@@ -64,6 +66,8 @@ export const QUERIES_MAP = {
   [RESOURCE_TYPES.EVENT_LISTING]: EventListing,
   [RESOURCE_TYPES.PERSON_PROFILE]: PersonProfile,
   [RESOURCE_TYPES.PRESS_RELEASE]: PressRelease,
+  [`${RESOURCE_TYPES.PRESS_RELEASE}--teaser` as const]: PressReleaseTeaser,
```

Comment:
Wondering if this should be an element '__teaser' instead of a modifier '--teaser'

Reply:
Issue was in listingPages.ts and incorrectly used the wrong pathing. Initially, placed 'press-releases' when it should have been 'news-releases'